ext/fetch/23_response.js

```diff
@@ -327,7 +323,6 @@ class Response {
       bodyWithType = extractBody(body);
     }
     initializeAResponse(this, init, bodyWithType);
-    this[_brand] = _brand;
```

Comment:
I think this should be first in order to help with monomorphism because webidl.createBrand(Response) will set _brand as the first property.